.github/workflows/tests.yml

```
@@ -0,0 +1,48 @@
+
+  sorting-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 5
+      
+      - name: Run sorting tests
+        run: |
+          ./tests/sort
+          echo 'example.com' >> list && ! ./tests/sort 
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)